index.js

```diff
@@ -2,10 +2,33 @@
 
 function keydown(fn: KeyboardEventHandler): KeyboardEventHandler {
   return function(event: KeyboardEvent) {
-    if (event.key === ' ' || event.key === 'Enter') {
+    const key = event.key
+    if (key === ' ' || key === 'Enter') {
       event.preventDefault()
       fn(event)
     }
+    if (key === 'ArrowRight' || key === 'ArrowLeft' || key === 'Home' || key === 'End') {
+      const target = event.currentTarget
+      if (!(target instanceof MarkdownButtonElement)) return
```

Comment:
In dotcom we have some custom buttons like save replies and suggested changes, can we perhaps introduce a data attribute to mark them as participants in this toolbar pattern as well?

Reply:
We can perhaps set this attribute on all the markdown buttons as well on `connectedCallback`, so then later we'll have an easier time querying for what should be consider as buttons in the navigation pattern.